code-studio/scripts/build-commands.js

```diff
@@ -295,3 +337,25 @@ function warnIfWrongNodeVersion() {
       console.log('You are using node ' + process.version + '. This build script expects ' + engines_node + '.');
   }
 }
+
+/**
+ * Wraps a function so that calling it with a set of arguments returns a new
+ * function with those arguments pre-bound - essentially a one-time curry.
+ * @param {!function} fn
+ * @returns {Function}
+ */
+function makeBuildCommand(fn) {
+  return function () {
+    return fn.bind.apply(fn, [undefined].concat([].slice.call(arguments)));
+  };
+}
```

Comment:
Code like this makes me want ES6 so bad :(
```
const makeBuildCommand = (fn, ...args) => () => fn(...args));
```
Could we also just do return _.curry(fn) here, and not have to trust that we got our custom currier right?